README.md

```diff
@@ -7,6 +7,25 @@
 
 Tooling to give you full type-safety around OpenAPI specs.
 
+- [Getting started](#getting-started)
+- [Philosophy](#philosophy)
+  - [Backend side](#backend-side)
+  - [Frontend side](#frontend-side)
+- [Configuration](#configuration)
+  - [Config Options](#config-options)
+    - [**filenamePrefix**](#filenameprefix)
+    - [**filenameCase**](#filenamecase)
+    - [**schemasFiles**](#schemasfiles)
+    - [**injected header**](#injected-header)
+  - [Example Config](#example-config)
+  - [Plugins](#plugins)
+    - [**generateSchemaTypes** (frontend/backend)](#generateschematypes-frontendbackend)
+    - [**generateFetchers** (frontend)](#generatefetchers-frontend)
+    - [**generateReactQueryComponents** (frontend)](#generatereactquerycomponents-frontend)
+    - [**generateReactQueryFunctions** (frontend)](#generatereactqueryfunctions-frontend)
+    - [**planned plugins**](#planned-plugins)
+- [Contributors ✨](#contributors-)
+
```

Comment:
Sorry, I’m not a fan of table of content 😅 This is actually already part of github UI
<img width="270" alt="image" src="https://user-images.githubusercontent.com/1761469/209525649-ce66d180-3c90-4430-8324-05536d69eea5.png">

Reply:
removed